include/text.php

```diff
@@ -951,10 +952,10 @@ function contact_block() {
 				dbesc(implode(",", $contacts)));
 
 			if (DBM::is_result($r)) {
-				$contacts = sprintf(tt('%d Contact','%d Contacts', $total),$total);
+				$contacts = sprintf(L10n::tt('%d Contact', '%d Contacts', $total), $total);
```

Comment:
Can you please replace all the
`sprintf(tt('...', '...', $arg), $arg)`
 constructs with
`L10n::tt('...', '...', $arg)`
?